rero_ils/modules/documents/utils.py

```diff
@@ -30,6 +30,91 @@
 from .dojson.contrib.marc21tojson.model import remove_trailing_punctuation
 from ...utils import get_i18n_supported_languages
 
+DOCUMENT_TYPES = {
```

Comment:
This can be added to the config.py file, no?

Reply:
Could be, but do we really like to put document facet correction specific definitions in the config?